pkg/util/kubernetes/apiserver/services.go

```diff
@@ -31,7 +58,12 @@ func (metaBundle *MetadataMapperBundle) mapServices(nodeName string, pods v1.Pod
 
 	for _, pod := range pods.Items {
 		if pod.Status.PodIP != "" {
-			podToIp[pod.Name] = pod.Status.PodIP
+			if _, ok := podToIp[pod.Namespace]; !ok {
+				podToIp[pod.Namespace] = make(map[string]string)
+			}
+			podToIp[pod.Namespace][pod.Name] = pod.Status.PodIP
```

Comment:
Is a continue here could remove the `else` block?

Reply:
Not a fan of `else` blocks are you.